### PR TITLE
Customer account navigation block container doesn't have  a name 

### PR DESCRIPTION
**Pull request by devamitbera**
- a name or as attribute
Add the suggestion  but it is only showing Label
-seems not good.So,add a condition to phtml.
It makes title only visible when it has childhtml

### Fixed Issues (if relevant)

1. magento/magento2#12892: Issue title

### Manual testing scenarios

1.Please open the Customer account link e.g. http://domain.com/customer/account
2.loggin as Customer  and then go to the Dashboard page
3.Now, you don't see any Section like **<strong></strong>** at the left section of customer account pages.
4.For case,I have added a **conditions to prevent to show the Only title/label* * whenever child does not have any html output By  adding **if ($block->getChildHtml()):**.It  is helpful for UI respective view.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Customer/view/frontend/layout/customer_account.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
             <block class="Magento\Framework\View\Element\Template" name="sidebar.main.account_nav" template="Magento_Theme::html/collapsible.phtml" before="-">
                 <arguments>
                     <argument name="block_css" xsi:type="string">account-nav</argument>
+                    <argument name="block_title" xsi:type="string">Account Sidebar</argument>
```
Comment (dmanners) [inline app/code/Magento/Customer/view/frontend/layout/customer_account.xml]: can you also include the `translate="true"` attribute here

**Review by dmanners — CHANGES_REQUESTED** [review]

File: app/code/Magento/Customer/view/frontend/layout/customer_account.xml (hunk 1 above)
Comment (devamitbera) [reply app/code/Magento/Customer/view/frontend/layout/customer_account.xml]: sure.I  am doing.

Thanks for feedback.

**Review by dmanners — APPROVED** [review]

**Comment by dmanners** [comment]
As discussed at mm18in I think that the original issue was to add the name attribute to the block (`name="sidebar.main.account_nav"`) which has already been done. I am not sure if we should actually add this title by default @okorshenko what would you think here?

**Comment by okorshenko** [comment]
The issue already fixed. Please see the comment. vhttps://github.com/magento/magento2/issues/12892#issuecomment-355815257

@devamitbera thank you for your contribution

**Pull request closed without merging**